hashicorp/dev-portal
# feat: adds generic branded callout block

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-ksadds-branded-hcp-callout-hashicorp.vercel.app/vault/tutorials) 🔎
- [Asana task](https://app.asana.com/0/1199634971449915/1202932567024649) 🎟️

## 🗒️ What

This PR adds a 'generic' variant to the branded callout, which will show when `product` isn't passed. This is what we will use for the hcp tutorials landing branded callout. 

## 📸 Design Screenshots
<img width="935" alt="Screen Shot 2022-09-08 at 2 33 41 PM" src="https://user-images.githubusercontent.com/36613477/189230030-d0207fb8-b210-4e37-8a1f-1e39de9a4e17.png">


## 🧪 Testing
- Visit this[ tutorials landing](https://dev-portal-git-ksadds-branded-hcp-callout-hashicorp.vercel.app/vault/tutorials) to see the styles (note that's a testing stub, as the hcp tutorials landing content doesn't have this block yet, [see related PR](https://github.com/hashicorp/learn-api-content-sync/pull/64)) 
- verify it matches the spec

## Before Merge
- [x] Remove comment for branded callout product
